ros2_rust.repos

```diff
@@ -2,15 +2,15 @@ repositories:
   ros2/common_interfaces:
     type: git
     url: https://github.com/ros2/common_interfaces.git
-    version: foxy
```

Comment:
Perhaps it'd make sense to have a separate `.repos` file only for Rolling, since interfaces packages may introduce incompatibilities between ROS 2 distros, @jhdcs what do you think?

Reply:
This sounds reasonable to me. I've updated the Dockerfile as well to reflect the change, and allow the user to select different `.repos` files for building.